command-expansion-server/src/app.ts

```diff
@@ -9,93 +9,94 @@ import { processDictionary } from "./packages/lib/CommandTypeCodegen.js";
 const PORT: number = parseInt(getEnv().PORT, 10) ?? 3000;
 
 const app: Application = express();
-app.use(bodyParser.json({ limit: '25mb' }));
+app.use(bodyParser.json({ limit: "25mb" }));
 
 DbExpansion.init();
 const db = DbExpansion.getDb();
 
-
 app.get("/", (req: Request, res: Response) => {
   res.send("Aerie Command Service");
 });
 
-app.put("/dictionary", async (req, res) => {
-  const dictionary = Buffer.from(req.body.input.dictionary, 'base64').toString();
-
-
-  console.log(`Dictionary received`);
-  const parsedDictionary = ampcs.parse(dictionary);
-  console.log(
-    `Dictionary parsed - version: ${parsedDictionary.header.version}, mission: ${parsedDictionary.header.mission_name}`
-  );
-  const commandDictionaryPath = await processDictionary(parsedDictionary);
-  console.log(`command-lib generated - path: ${commandDictionaryPath}`);
-
-  const sqlExpression = `
-    insert into command_dictionary (command_types, mission, version)
-    values ($1, $2, $3)
-    on conflict (mission, version) do update
-    set command_types = $1
-    returning id;
-  `;
-
-  const { rows } = await db.query(sqlExpression, [
-    commandDictionaryPath,
-    parsedDictionary.header.mission_name,
-    parsedDictionary.header.version,
-  ]);
-
-
-
-  if (rows.length < 0) {
-    console.error(`POST /dictionary: No command dictionary was updated in the database`);
-    res.status(500).send(`POST /dictionary: No command dictionary was updated in the database`);
-    return;
+app.post("/dictionary", async (req, res) => {
+  try {
```

Comment:
We can remove the try-catch here as we have a global error handler down at lines 97-100